pkg/controllers/provisioning/scheduling/node.go

```diff
@@ -91,7 +96,13 @@ func (n *Node) Add(pod *v1.Pod) error {
 
 	// Check instance type combinations
 	requests := resources.Merge(n.requests, resources.RequestsForPods(pod))
-	instanceTypes := filterInstanceTypes(n.InstanceTypeOptions, nodeRequirements, requests)
+	// determine the number of volumes that will be mounted if the pod schedules
+	mountedVolumeCount, err := n.volumeUsage.Validate(pod)
```

Comment:
We call this variable `mountedVolumeCount` and `volumesPerNode` in different parts of the code. Can we simplify to a single concept to minimize cognitive load for the reader? Perhaps just `volumeCount` with "mounted" and "node" implied.

Reply:
The results of `Validate()` is called `mountedVolumeCount` both times it's used.  This is the number of volumes that would be mounted on the node if the pod were added.  I think renaming this to volumeCount is slightly ess clear.  I renamed `volumesPerNode` to `mountedVolumeCount`.